heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -556,5 +536,63 @@ specify for `limit` should be below the configured ulimit.
 
 The timezone for the scheduler. By default the scheduler uses localtime.
 
+[float]
+[[monitor-watch-poll-file]]
+==== `watch.poll_file`
+
+experimental[]
+
+THIS EXPERIMENTAL FEATURE IS NOW DEPRECATED. IT WILL BE REMOVED IN A FUTURE
```

Comment:
You should replace the experimental[] tag with deprecated (see https://github.com/elastic/docs#section-notifications). Within the tag itself, note that you can add details to provide more info about the deprecation.